### ISSUE TITLE
JSdoc doesn't match implementation

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/projectile-motion#26]
Related to #22 (high-level review).

In many places the JSdoc doesn't match the implementation.  Two examples from `LabSecondPanel`:

```js
  /**
   * @param {ProjectileMotionModel} model
   * @param {CustomizeDialog} customizeDialog - panel linked to customize button
   * @constructor
   */
  function LabSecondPanel( projectileMotionLabModel, options ) {

...

    /**
     * Auxiliary function that creates vbox for a parameter label and readouts
     * @param {string} label
     * @param {Property.<number>} property - the property that is set and linked to
     * @param {Object} range, range has keys min and max
     * @returns {VBox}
     * @private
     */
    function createParameterControlBox( labelString, unitsString, property, range ) {
```

When you make changes to function signatures, be sure to update the corresponding `@param` annotations.  And use valid type expressions.

**Comment by pixelzoom** [comment]
Btw... This type of mismatch is very obvious in IntelliJ IDEA.  For example, here's a screenshot showing the above example:

![screenshot_229](https://cloud.githubusercontent.com/assets/3046552/25367808/71cd2c02-2935-11e7-990d-c4d82bd4bf29.png)

**Comment by andrea-phet** [comment]
Assigning to @pixelzoom to review.

**Comment by andrea-phet** [comment]
Actually, I found a couple more mismatches, reassigning back to myself.

**Comment by pixelzoom** [comment]
If you're using IDEA, it will show you the mismatches.  If you're using Sublime, I'm not sure what your options are - ask other Sublime uses on the Skype developer thread.

**Comment by andrea-phet** [comment]
Assigning to @pixelzoom to review and close.

**Comment by pixelzoom** [comment]
I skimmed through the code, documentation of function parameters looks generally good.  I did notice that documentation was consistently missing for `options` parameters.  While `options` is indeed an optional parameter, it still requires documentation, of the form:

```js
@param {Object} [options]
```

I've added documentation for `options` in the above commit.  @andrea-phet let me know if you have any questions about this, and feel free to close this issue.
